core/spec/models/spree/product_spec.rb

```diff
@@ -485,4 +485,28 @@ def build_option_type_with_values(name, values)
     product = Spree::Product.new
     expect(product.master.is_master).to be true
   end
+
+  context '#discontinue!' do
+    let(:product) { create(:product, sku: 'a-sku') }
+
+    it "sets the discontinued" do
+      product.discontinue!
+      product.reload
+      expect(product.discontinued?).to be(true)
+    end
+  end
+
+  context "#discontinued?" do
+    let(:product_live) { build(:product, sku: 'a-sku') }
+    it "should be false" do
+
+      expect(product_live.discontinued?).to be(false)
+    end
+
+    let(:product_discontinued) { build(:product, sku: 'a-sku',
+                                       discontinue_on: Time.now - 1.day)  }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 507, 75 should be on its own line.<br>Unnecessary spacing detected.